packages/squiggle-lang/__tests__/GenericDist/GenericDistSparkline_Test.res

```diff
@@ -0,0 +1,35 @@
+open Jest
+open Expect
+
+let env: DistributionOperation.env = {
```

Comment:
not needed in this file (unless you implement my comment about actually using an `~env` in the sparkline function.